joomla/joomla-cms
# [4.0] Template header icon color

Pull Request for Issue #14763 .

### Summary of Changes
Change template header title icon color to white, fixing a11y. Add aria-hidden="true" attrib.

### Testing Instructions
Apply patch. Check header (#14763 )

![icon](https://cloud.githubusercontent.com/assets/2803503/24080922/be0f6d34-0ca0-11e7-965e-334f1f44c076.png)
